sleap_nn/training/model_trainer.py

```
@@ -38,6 +38,24 @@
     ConvNeXt_Large_Weights,
 )
 
+import sleap_io as sio
```

Ruff

Redefinition of unused `sio` from line 11

Remove definition: `sio`

(F811)